packages/gatsby-theme-docs/src/layouts/internals/layout-page-navigation.js

```diff
@@ -34,13 +34,6 @@ const GridContainer = styled.div`
   @media screen and (${designSystem.dimensions.viewports.largeTablet}) {
     display: block;
     grid-area: page-navigation;
-    width: ${designSystem.dimensions.widths.pageNavigation};
-  }
-  @media screen and (${designSystem.dimensions.viewports.laptop}) {
-    width: ${designSystem.dimensions.widths.pageNavigationSmall};
-  }
-  @media screen and (${designSystem.dimensions.viewports.desktop}) {
-    width: ${designSystem.dimensions.widths.pageNavigation};
```

Comment:
Please what is the reason these are being removed?

Reply:
When digging into the whole grid logic and how space is used I realized that on tablet and laptop the constraint is redundant because the space is already constrained by the grid. 

On desktop, I found that it's not necessary to force a smaller navigation when the space is available - i find it easier on the eye and visually nicer if the entries are one line:

![image](https://user-images.githubusercontent.com/4006139/86398232-8e39cd80-bca5-11ea-8b25-7b9e1a1fb58c.png)